Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/Exporters/IUnityProjectExporter.cs

```diff
@@ -40,10 +40,10 @@ public interface IProjectExporter
         void ExportSolution(UnityProjectInfo unityProjectInfo, MSBuildToolsConfig config);
 
         /// <summary>
-        /// Generates the Directory.Build.props file that is expected to be used by both generated and non-generated projects alike.
+        /// Creates an exporter for the commom MSBuild file that is expected to be used by both generated and non-generated projects alike.
```

Comment:
>is expected to be used by both generated and non-generated projects alike [](start = 65, length = 73)

nit: I don't know what this description means. I do not know what a generated vs non-generated project is

Reply:
Updating.